adapters/adnuntius/adnuntiustest/supplemental/send-header-information.json

```diff
@@ -0,0 +1,108 @@
+{
+    "mockBidRequest": {
+        "id": "test-request-id",
+        "user": {
+            "id": "1kjh3429kjh295jkl",
+            "ext": {
+                "consent": "CONSENT_STRING"
+            }
+        },
+        "device": {
+            "ip": "158.174.81.143",
+            "ua": "Mozilla/5.0 (Linux; Android 8.0; Pixel 2 Build/OPD3.170816.012) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Mobile Safari/537.36"
+        },
+        "imp": [
+            {
+                "id": "test-imp-id",
+                "banner": {
+                    "format": [
+                        {
+                            "w": 300,
+                            "h": 250
+                        },
+                        {
+                            "w": 300,
+                            "h": 600
+                        }
+                    ]
+                },
+                "ext": {
+                    "bidder": {
+                        "auId": "123"
+                    }
+                }
+            }
+        ]
+    },
+
+    "httpCalls": [
+        {
+            "expectedRequest": {
+                "uri": "http://whatever.url?format=json&tzo=0",
```

Comment:
Please add a check for expected headers under under `httpCalls.expectedRequest`:

```
"headers": {
          "Content-Type": [
            "application/json;charset=utf-8"
          ],
          "Accept": [
            "application/json"
          ],
          "X-Forwarded-For": [
            "158.174.81.143"
          ],
          "User-Agent": [
            "Mozilla/5.0 (Linux; Android 8.0; Pixel 2 Build/OPD3.170816.012) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.93 Mobile Safari/537.36"
          ]
        },

```

Reply:
```
"httpCalls": [
    {
      "expectedRequest": {
           "headers": {
                .....   
           },
          "uri": "http://whatever.url?format=json&tzo=0",
```

Reply:
Thanks! I was looking for how to add that but I couldn't really find it anywhere. I've now updated the coed.